### PR TITLE
Fixed strange error at 4am and various small improvements

### PR DESCRIPTION
**Pull request by LukeTheWalker**

### CODE REVIEW

File: albo.py [hunk 1]
```diff
@@ -25,14 +26,33 @@ def main ():
...
+    #avoid channel flooding in case of error
+    if len(ids_to_parse) > 10 and '--force-unsafe' not in sys.argv:
+        ids_to_parse = []
+        print("Troppi annunci da parsare, usare --force-unsafe per forzare l'invio")
+
     for id in ids_to_parse:
-        tr = table.find('td', text=id).parent
+        td = table.find('td', text=id)
+
+        if (td is None):
```
Comment (Helias) [inline albo.py:49]:
```suggestion
        if td is None:
```

File: albo.py [hunk 2]
```diff
@@ -25,14 +26,33 @@ def main ():
...
+            tg_bot.send_debug_messages("ID non trovato: " + str(id) + "\n" + "Probabilmente lo hanno saltato")
+            if id in cached_announcements:
+                cached_announcements.remove(id)
+            continue
```
Comment (Helias) [inline albo.py:53]: ???

File: utils/tg.py [hunk 3]
```diff
@@ -69,10 +69,10 @@ def send_telegram_announcements(self, pdf_links: List[str], message: str) -> Non
             self.send_multiple_telegram_attachments(pdf_links)
             self.send_telegram_message(message)
     
-    def send_documents_error_message(self, err: ValueError) -> None:
+    def send_documents_error_message(self, err: ValueError, id: int) -> None:
         decoded_url = json.dumps(unquote(err.args[1]))
         document_list = re.findall(r"https?:.+?(?=\\|\")", decoded_url)
-        error_string = "```response sent: " + err.args[0] + "\n```" + "related documents:\n" + escape_char('\n'.join(document_list))
+        error_string = "id:" + str(id) + "\n" + "``` response sent: " + err.args[0] + "\n```" + "related documents:\n" + escape_char('\n'.join(document_list))
```
Comment (Helias) [inline utils/tg.py:75]:
```suggestion
        error_string = "id: " + str(id) + "\n" + "``` response sent: " + err.args[0] + "\n```" + "related documents:\n" + escape_char('\n'.join(document_list))
```

File: albo.py (hunk 2 above)
Comment (LukeTheWalker) [reply albo.py:53]: Sometimes it happens that a particular number is not used in sequential order (e.g. 934, 908), previously we just saved its id in the cached announcement file and kept searching for the aforementioned id, without any success. With this check the bot basically ignores that specific id number, sends a one-time debug message to the devs, in order to make sure it was not mistakenly discarded, and goes on as normal

**Review by Helias — APPROVED** [review]

**Pull request merged**